pkg/csi/service/wcp/controller.go

```diff
@@ -1697,6 +1735,23 @@ func (c *controller) CreateSnapshot(ctx context.Context, req *csi.CreateSnapshot
 	return resp, err
 }
 
+func patchCNSVolumeInfoWithRetry(ctx context.Context, volumeID string, patchBytes []byte) error {
+	log := logger.GetLogger(ctx)
+	retries := 0
+	for {
+		retries++
+		err := volumeInfoService.PatchVolumeInfo(ctx, volumeID, patchBytes)
+		if err != nil && retries == 5 {
```

Comment:
how did we decide 5 here? maybe make this configurable, for now a const variable should do.